vignettes/articles/03-run-stress-test.Rmd

```diff
@@ -88,6 +88,9 @@ stress_test_arguments
 ### Understanding the input arguments
 The meaning of each of the detail arguments is as follows:
 
+* `baseline_scenario` holding the name of the baseline scenario.
+* `shock_scenario` holding the name of the scenario that companies are shocked 
```

Comment:
similar as above:
`shock_scenario` holds the name of the scenario that companies are forced to transition to via a late and sudden shock. We use an ambitious transition scenario as the shock scenario.

Reply:
done